root/static/scripts/common/immutable-entities.js

```diff
@@ -42,7 +42,7 @@ export const reduceArtistCredit =
  */
 export function reduceArtistCreditNames(
   names: $ReadOnlyArray<ArtistCreditNameT>,
-  dropFinalJoinPhrase?: false,
+  dropFinalJoinPhrase?: boolean = false,
```

Comment:
Whoops, I think this was a bug from one of my earlier changes. Still getting used to Flow...